.changeset/friendly-clocks-act.md

```diff
@@ -0,0 +1,13 @@
+---
+'create-astro': patch
+'@astrojs/telemetry': patch
+'astro': patch
+---
+
+Update preferred-pm and which-pm-runs packages as the latest versions now support Bun.
+
+Standardized variable naming by changing all instances of pkgManager to packageManager to ensure consistent wording throughout the codebase.
+
+`bunx astro add` command correctly uses bun.
+
+Telemetry notice returns package manager.
```

Comment:
I think we could keep this a bit simpler. The changelog here is usually user-facing so we can simplify it, e.g.
```suggestion
Support detecting Bun when logging messages with package manager information
```